apps/web/src/pages/ApplicantDashboardPage/components/QualifiedRecruitments/QualifiedRecruitmentCard.tsx

```diff
@@ -0,0 +1,417 @@
+import * as React from "react";
+import { notEmpty } from "@gc-digital-talent/helpers";
+import { getLocale } from "@gc-digital-talent/i18n";
+import {
+  Dialog,
+  Button,
+  Chip,
+  Chips,
+  Heading,
+  HeadingProps,
+} from "@gc-digital-talent/ui";
+import {
+  ExclamationCircleIcon,
+  CheckCircleIcon,
+  LockClosedIcon,
+} from "@heroicons/react/24/solid";
+import { useIntl } from "react-intl";
+import {
+  PoolCandidate,
+  Skill,
+  useChangeApplicationSuspendedAtMutation,
+} from "~/api/generated";
+import ApplicationActions from "~/pages/Applications/MyApplicationsPage/components/ApplicationCard/ApplicationActions";
+import {
+  isDraft,
+  isExpired,
+  isPlaced,
+} from "~/pages/Applications/MyApplicationsPage/components/ApplicationCard/utils";
+import { getFullPoolAdvertisementTitleHtml } from "~/utils/poolUtils";
+import { toast } from "react-toastify";
+import { FormProvider, useForm } from "react-hook-form";
+
+interface QualifiedRecruitmentDialogProps {
+  openDialogLabel: string;
+  title: string;
+  primaryBodyText: string;
+  secondaryBodyText: string;
+  closeButtonLabel: string;
+  id: string;
+  isSuspended: boolean;
+}
+
+type FormValues = {
+  isSuspended: boolean;
+};
+
+// TODO: Should the dialog component be pulled out into another file?
+const QualifiedRecruitmentDialog = ({
+  openDialogLabel,
+  primaryBodyText,
+  secondaryBodyText,
+  closeButtonLabel,
+  title,
+  id,
+  isSuspended,
+}: QualifiedRecruitmentDialogProps) => {
+  const intl = useIntl();
+  const [, executeMutation] = useChangeApplicationSuspendedAtMutation();
+  const [isOpen, setIsOpen] = React.useState<boolean>(false);
+
+  const methods = useForm<FormValues>({ defaultValues: { isSuspended } });
+
+  const {
+    handleSubmit,
+    formState: { isSubmitting },
+  } = methods;
+
+  const updateSuspendedAtStatus = async () => {
+    await executeMutation({
+      id,
+      isSuspended: !isSuspended,
+    })
+      .then((res) => {
+        if (!res.error) {
+          setIsOpen(false);
+        }
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "INSERT ERROR CONTENT HERE.",
```

Comment:
Do we have content to be added here or are we waiting on it?

Reply:
Alright, I get some text and added it in! [3b2a714](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5988/commits/3b2a714adac9357471f4b690d6dfa5bffcdf8db0)